eta-meeting-organizer-frontend/src/app/auth/components/auth.component.ts

```diff
@@ -0,0 +1,45 @@
+import {Component, OnDestroy } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { Subscription } from 'rxjs';
+import { AuthService } from '~/app/shared/services/auth.service';
+
+@Component({
+  selector: 'app-auth',
+  styles: [
+    `
+    #row {
+      min-height: calc(100vh - 60px);
```

Comment:
ez a header meg a footer közé szoritás miatt van? ha igen kerüljön ki main layouthoz

Reply:
Ez több komponensnél is fenn áll, hogy css-ben állítjuk be a magasságot. Kipróbáltam egy csomó variációt, de sosem tölti ki a külső div a teljes képernyőt.